peril.settings.json

```diff
@@ -10,7 +10,7 @@
     // Jira integration
     "pull_request": ["org/allPRs.ts", "org/jira/pr.ts", "org/mobilePRsForbidForks.ts"],
     "pull_request.closed": ["org/closedPRs.ts", "org/jira/pr.ts"],
-    "pull_request.opened": "org/addPatchLabel.ts",
+    "pull_request.opened": ["org/openedPR.ts", "org/addPatchLabel.ts"],
```

Comment:
Does this reference a file that you were considering but then removed/renamed? It doesn't look like it exists in `master` or in this diff.

Reply:
oooo yep you are very much correct 😬 good catch